.evg.yml

```diff
@@ -55,6 +56,7 @@ buildvariants:
       - name: "test-smoke"
       - name: "publish-maven"
       - name: "download-center-update"
```

Comment:
Let's put this one last because it is easy to manually update it if it fails, but we would still need the trace and ssdlc-artifacts to be in